pt-BR/mail/chrome/messenger/preferences/downloads.dtd

```diff
@@ -1,24 +1,24 @@
-<!ENTITY askOnSave.label              "Attachments Folder">
-<!ENTITY alwaysAsk.label              "Ask me where to save every file">
+<!ENTITY askOnSave.label              "Pasta de anexos">
+<!ENTITY alwaysAsk.label              "Perguntar onde salvar cada arquivo">
 <!ENTITY alwaysAsk.accesskey          "A">
-<!ENTITY neverAsk.label               "Save all attachments to this folder:">
+<!ENTITY neverAsk.label               "Salvar todos os anexos nesta pasta:">
 <!ENTITY neverAsk.accesskey           "S">
-<!ENTITY desktop.label                "Desktop">
-<!ENTITY downloads.label              "My Downloads">
-<!ENTITY chooseFolderWin.label        "Browse...">
+<!ENTITY desktop.label                "Área de Trabalho">
+<!ENTITY downloads.label              "Downloads">
+<!ENTITY chooseFolderWin.label        "Procurar...">
 <!ENTITY chooseFolderWin.accesskey    "B">
-<!ENTITY chooseFolderMac.label        "Choose...">
+<!ENTITY chooseFolderMac.label        "Escolher...">
 <!ENTITY chooseFolderMac.accesskey    "C">
 
 
-<!ENTITY fileTypes.label              "Download Actions">
-<!ENTITY configureActions.label       "View &amp; Edit Actions...">
+<!ENTITY fileTypes.label              "Ações de Download">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY fileTypes.label              "Ações para baixar">